docs/explanations/how_to_interpret_output_of_fix_execute_api.md

```diff
@@ -0,0 +1,115 @@
+# How to interpret the output of the fix execute API endpoint
+
+In the Pro Client version 29, we introduce two distinct endpoints:
+
+* `u.pro.security.fix.cve.execute.v1`
+* `u.pro.security.fix.usn.execute.v1`
+
+These endpoints can be used to fix a list of [CVEs/USNs](cves_and_usns_explained.md).
+When using the CVE endpoint, the expected output is as follows:
```

Comment:
Let's put a heading between lines 8 and 9 with something like "Using the CVE endpoint"